docs/core/install/includes/linux-release-wait.md

```diff
@@ -1,10 +1,10 @@
 ---
 author: adegeo
 ms.author: adegeo
-ms.date: 11/14/2023
+ms.date: 11/04/2024
 ms.topic: include
 ms.custom: linux-related-content
 ---
 
 > [!IMPORTANT]
-> .NET 8 was released on November 14, 2023. It may take time for the packages to appear in the package manager feeds.
+> .NET 9 was released on November 12, 2024. It may take time for the packages to appear in the package manager feeds, or for your specific Linux distribution to include it.
```

Comment:
Ignore this.. This is in preparation for the next PR that will fully integrate .NET 9. There's always a huge lag between release and actual availability so I have this alert posted to all of the install articles for a month or so.